src/main/java/controllers/ExceptionController.java

```diff
@@ -0,0 +1,4 @@
+package controllers;
+
```

Comment:
hierin mis ik de @ControllerAdvice van boven de klasse. ook zet je hier de logica van de ExceptionHnadler in. bij voorbeeld @ExceptionHandler(value = RecordNotFoundException.class)
    public ResponseEntity<Object> exception(RecordNotFoundException exception) {

        return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);

    }